distmaker/dists/common.sh

```diff
@@ -82,7 +82,7 @@ function dm_install_core() {
   local repo="$1"
   local to="$2"
 
-  for dir in ang css i js PEAR templates bin CRM api extern Reports install mixin settings Civi partials release-notes xml setup sql/civicrm_data ; do
+  for dir in ang css i js PEAR templates bin CRM api extern Reports install managed mixin settings Civi partials release-notes xml setup sql/civicrm_data ; do
```

Comment:
Nice. :+1:  That one is easy to forget.

Reply:
@totten I looked for other places where we hard-code root directory names but didn't see any more. Unless you know of some.